atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/Blacklist.java

```diff
@@ -70,7 +70,7 @@ private boolean isHostHealthy(CassandraClientPoolingContainer container) {
             container.runWithPooledResource(CassandraUtils.getValidatePartitioner(config));
             return true;
         } catch (Exception e) {
-            log.warn("We tried to add blacklisted host '{}' back into the pool, but got an exception"
+            log.debug("We tried to add blacklisted host '{}' back into the pool, but got an exception"
```

Comment:
Very often this is `TimedOutException` or various Thrift-based things, though I don't know if this one is still useful for debugging?

Reply:
I would put this to info, as otherwise the information will just be lost at a point when we're trying to diagnose any blacklisting issues.

However, I would like the message to be improved - what does it mean that we're distrusting it further? When will we next attempt to contact this host? This could potentially be done as a separate, small(?) PR.